blockchain/blockdao/blockdao.go

```diff
@@ -719,9 +720,13 @@ func (dao *blockDAO) putBlock(blk *block.Block) error {
 	}
 	batchForBlock := batch.NewBatch()
 	hash := blk.HashBlock()
+	heightKey := heightKey(blkHeight)
 	batchForBlock.Put(blockHeaderNS, hash[:], serHeader, "failed to put block header")
 	batchForBlock.Put(blockBodyNS, hash[:], serBody, "failed to put block body")
 	batchForBlock.Put(blockFooterNS, hash[:], serFooter, "failed to put block footer")
+	if sysLog := action.SystemLog(blk.Receipts); len(sysLog) != 0 {
+		batchForBlock.Put(systemLogNS, heightKey, sysLog.Serialize(), "failed to put system log")
+	}
```

Comment:
I thought we only write system log only if the feature is turned on.

Reply:
yeah, let's discuss what's the best plan